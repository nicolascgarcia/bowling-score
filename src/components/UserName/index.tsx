import styled from 'styled-components';

export default styled.h1`
	font-size: 26px;
	font-weight: bold;
	color: white;
	text-overflow: ellipsis;
	white-space: nowrap;
	overflow: hidden;
`;
