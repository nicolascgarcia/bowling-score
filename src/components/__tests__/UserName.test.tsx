/* eslint-disable import/no-extraneous-dependencies */
import renderer from 'react-test-renderer';
import UserName from '../UserName';

describe('<UserName/>', () => {
	it('should render UserName', () => {
		const component = renderer.create(<UserName />);
		const tree = component.toJSON();
		expect(tree).toMatchSnapshot();
	});
});
