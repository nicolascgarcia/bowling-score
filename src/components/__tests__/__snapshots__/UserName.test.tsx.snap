// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<UserName/> should render UserName 1`] = `
<h1
  className="sc-bczRLJ hBREsu"
/>
`;
